esp/Gps/README.md

```diff
@@ -0,0 +1,10 @@
+# matty
+Conection joint encoder to esp board:
+
+  black   - GND
+
+  red     - 3V3
+
+  white   - IO5 
+
+![Conection joint encoder - esp board](../../Images/encoder-esp.jpg)
```

Comment:
be aware of linux/windows world. It looks like the images should be with small "i" (i.e. I do not see it)

Reply:
p.s. is not the name a bit misleading? or it is really some kind of encoder?